### PR TITLE
[WIP] Add playground pages for Successfulness types

### PR DESCRIPTION
**Pull request by JonathanHoffman**
This Pull Request supports #119. 

Brentley is working on implementing successfulness types, and asked me to help write playground pages for them!
#### Features
- Playground pages for both new types.
  - [x] Write playground page for `Successfulness`
  - [ ] Write playground page for `OutcomeWithSuccessfulness`
#### Mentions

@brentleyjones

### CODE REVIEW

**Comment by brentleyjones** [comment]
You have some old commits of mine here. I would cherry pick your changes over the latest one to make sure you only have your commits listed in this PR.

File: DiceKit.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -364,7 +376,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0700;
-				LastUpgradeCheck = 0700;
+				LastUpgradeCheck = 0720;
```
Comment (brentleyjones) [inline DiceKit.xcodeproj/project.pbxproj]: Rebasing/cherry picking should remove the need for this.

File: TomeOfKnowledge.playground/contents.xcplayground [hunk 2]
```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<playground version='6.0' target-platform='osx' requires-full-environment='true' display-mode='rendered'>
+<playground version='6.0' target-platform='osx' display-mode='raw'>
```
Comment (brentleyjones) [inline TomeOfKnowledge.playground/contents.xcplayground:2]: Be sure to not check in `raw` mode.

Comment (JonathanHoffman) [inline TomeOfKnowledge.playground/contents.xcplayground:2]: I should have fixed this last night. Oops!

**Pull request closed without merging**